src/crons/EpicGames.ts

```diff
@@ -0,0 +1,201 @@
+import { MessageEmbed } from 'discord.js';
+import got from 'got';
+
+import { Cron, findTextChannelByName } from '../framework';
+
+const dateFmtOptions: Intl.DateTimeFormatOptions = {
+  year: 'numeric',
+  month: 'long',
+  day: 'numeric',
+  hour: 'numeric',
+  minute: 'numeric',
+};
+
+export default new Cron({
+  enabled: true,
+  name: 'EpicGames',
+  description:
+    'Vérifie toutes les heures si Epic Games offre un jeu (promotion gratuite) et alerte dans #jeux',
+  schedule: '0 * * * *',
```

Comment:
```suggestion
  schedule: '0 18 * * *',
```

This is for example every day at 18h00. Is there a usual time when EG publishes new offers?

Reply:
16h00 seems to me a good average.